### ISSUE TITLE
Font Replacement

### ISSUE THREAD

**Issue by Zet0rz** [Zet0rz/nzombies#307]
So turns out the font we're currently using does [not allow being used in games/apps/software](http://www.dafont.com/dk-umbilical-noose.font) which means we'll probably wanna replace it if we can find a good alternative. So you guys got some ideas of what fonts we can use?

I found some alternatives here:
- [Haunt AOE](http://www.dafont.com/haunt-aoe.font?text=%26%238734%3B&back=theme)
- [Anarchy](http://www.dafont.com/anarchy2.font?text=%26%238734%3B&psize=l&back=theme)

We'll probably want some bloody-like font that has at least all numbers and letters and a few symbols. A ∞-sign would also be nice, but not needed if I can just make a png of it instead (I have something planned for the easter egg finales).

**Comment by lolleko** [comment]
Lol. All assets ripped from COD:WaW, but the font concerns you :D Anyways the first one is better the second one is to transparent.

**Comment by Zet0rz** [comment]
I don't actually know why I bothered with it, I guess I just noticed it on the license and thought about it lol. If no good alternative shows up, we could probably still stick with what we have, but I kinda just feel like there's something different about using a small thing from a random guy that specifically says not to use it for this, than to rip from major companies for recreational purposes ... if that made sense

**Comment by minin43** [comment]
Wait I'm confused. What's the issue?

**Comment by Zet0rz** [comment]
I'll keep this open in case we get some good alternatives, but I won't be changing it if we don't.

**Comment by tylerpadgett** [comment]
Some people don't see the font, as it has spaces in it. (Idk, all of the people have had this problem, even after subscribing to the workshop addon). I like the Anarchy, though.

**Comment by func-octo** [comment]
Edo SZ is the closest I dould find to COD: [http://www.dafont.com/edo.font?text=NAZI+ZOMBIES](url)
You Murderer BB  is a good one with more characters: [http://www.fontspace.com/blambot/youmurderer-bb](url)
Rainy Day Vandal is a close alternative: [http://www.fontspace.com/kc-fonts/rainy-day-vandal](url)

Idk abour you guys but Edo SZ is awesome, minus the fact that it's an all caps font. That could be fixed with font sizing if you really wanted to. Anarchy is alright but it doesn't have good tally mark symbols.

**Comment by minin43** [comment]
I already spoke to Zet0r about it, we are OKAY to continue using the font.

Personally, changing the font is a small detail but would have drastic effects on players who have never played the gamemode before and are expecting a fully authentic experience. Changing the font away from the original would take away from that authenticity.

I'm against any change.

**Comment by func-octo** [comment]
> I'm against any change.

I agree with you on this, just wanted to help out :)

**Comment by Zet0rz** [comment]
> the original

If you're talking about the original CoD one, that's just like a simple Arial or something:
![](http://pad1.whstatic.com/images/6/66/Survive-in-Cod-Black-Ops-Zombies-%28Kino-Der-Toten%29-Step-5.jpg)

lol

**Comment by minin43** [comment]
No, the stylized one we have now is the title font used in all the previous games. I like the stylized font just about everywhere (although it can be a bit hard to read in some scenarios), so again, against any change.

**Comment by lolleko** [comment]
I like the arial one tbh :D Readability should be more important than style.
